Index: createtorrent-1.1.4/configure.in
===================================================================
--- createtorrent-1.1.4/configure.in
+++ createtorrent-1.1.4/configure.in	2008-06-09 11:54:21.000000000 +0300
@@ -2,7 +2,7 @@
 AM_CONFIG_HEADER(config.h)
 AM_INIT_AUTOMAKE(createtorrent,1.1.4)
 AC_PROG_CC
-AC_CHECK_LIB([ssl],[SHA1],,[AC_MSG_ERROR([error, OpenSSL required])])
+AC_CHECK_LIB([crypto],[SHA1],,[AC_MSG_ERROR([error, OpenSSL required])])
 AC_HEADER_DIRENT
 AC_HEADER_STDC
 AC_CHECK_HEADER([openssl/sha.h],,[AC_MSG_ERROR([Error, OpenSSL header file sha.h not found.])])
